docker-compose.yml

```diff
@@ -251,7 +251,7 @@ services:
 
   reviews:
     image: quay.io/centerforopenscience/ember-osf-reviews:develop-development-local
-    command: /bin/bash -c "yarn --pure-lockfile --no-progress --no-emoji && ./node_modules/bower/bin/bower install --allow-root --config.interactive=false && ./node_modules/ember-cli/bin/ember serve --host 0.0.0.0 --port 4400"
+    command: /bin/bash -c "yarn --pure-lockfile --no-progress --no-emoji && ./node_modules/ember-cli/bin/ember serve --host 0.0.0.0 --port 4400"
```

Comment:
Worth commenting but probably not worth changing. `yarn run ember server ...` will work as well.